js/app.js

```diff
@@ -0,0 +1,176 @@
+// Array to store currency rates
+let currencyRates = [];
+
+// function  to show message
+function displayMessage(message, isSuccess = true) {
+  const messageDiv = document.getElementById("message");
+  messageDiv.textContent = message;
+
+  // Setting color if succes it will be green if error - red
+  if (isSuccess) {
+    messageDiv.style.color = "green";
+  } else {
+    messageDiv.style.color = "red";
+  }
+
+  messageDiv.style.fontWeight = "bold";
+
+  // after 3 seconds removing message
+  setTimeout(function () {
+    messageDiv.textContent = "";
+  }, 3000);
+}
+
+// Function for adding a new exchange rate
+function handleAddRateForm(event) {
+  event.preventDefault(); // Prevent the default form submission behavior
+
+  // Get the value of the base currency from the input field
+  const baseCurrency = document
+    .getElementById("base-currency")
+    .value.trim()
```

Comment:
Be careful; there are too many `.value` here. It should be 
```js
document
    .getElementById("base-currency")
    .value.trim()
    .toUpperCase();
```
This comment is also applicable to the other functions below.